src/wp-admin/about.php

```diff
@@ -516,7 +516,7 @@
 					printf(
 						/* translators: 1: Learn WordPress link. */
 						__( 'Explore <a href="%s">learn.wordpress.org</a> for tutorial videos, online workshops, courses, and lesson plans for Meetup organizers, including new features in WordPress.' ),
-						'https://learn.wordpress.org/'
+						esc_url( __( 'https://learn.wordpress.org/' ) )
```

Comment:
Is there any reason to make this link translatable?

Reply:
Thanks @audrasjb for the review.
I think it shouldn't be translatable. So don't think the __ should be used here.

Reply:
https://github.com/WordPress/wordpress-develop/blob/3d5454875ba02936d20ef5fc67d5cb55d7eefebe/src/wp-admin/about.php#L219

Reply:
To my knowledge, the Learn WordPress site is only available in the the default, `en_US` locale. I agree that there's no need to mark this string for translation at this time.

Reply:
https://learn.wordpress.org/?locale=es_ES is available and _might_ be why this is translatable. Would defer to @SergeyBiryukov and others on this though.

Reply:
Ah! Thanks for finding that, I was trying es.learn.wordpress.org. The ticket attached to this PR has been closed, though, as the original intent was determined a `wontfix`. A new ticket to translate this URL (perhaps after confirmation from someone more familiar with translations) would be great!